rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -0,0 +1,104 @@
+:_content-type: REFERENCE
+[id="rosa-release-notes"]
+= {product-title} release notes
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-release-notes
+
+toc::[]
+
+{product-title} ROSA is a fully-managed, turnkey application platform that allows you to focus on delivering value to your customers by building and deploying applications. Red Hat and AWS Site reliability engineering (SRE) experts manage the underlying platform so you do not have to worry about the complexity of infrastructure management. ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to further accelerate the building and delivering of differentiating experiences to your customers.
+
+[id="about-this-release-section_{context}"]
+== About this release
+
+{product-title} (ROSA) https://access.redhat.com/errata/RHEA-2022:2252[RHEA-2022:2252] is now available. This release uses Kubernetes 1.23 with CRI-O runtime. New features, changes, and known issues that pertain to ROSA 4.10 are included in this topic.
+
+ROSA clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for ROSA, you can deploy OpenShift clusters to either on-premise or cloud environments.
+
+[id="about-this-release_{context}"]
+== ROSA layered and dependent component support and compatibility
+The scope of support for layered and dependent components of ROSA changes independently of the ROSA version. To determine the current support status and compatibility for an add-on, refer to its release notes. For more information, see the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle].
+
+[id="rosa-new-features"]
+== New features and enhancements
```

Comment:
Aside from the one xref link on line 53, this new features section seems like it could be a separate module?
I would recommend splitting this into its own module and moving the link to an Additional resources section at the end of the assembly.

If this is staying as is, I think that the ID needs to be updated so that it's consistent with other IDs (it's missing `{context}`).

Reply:
Good point. Updated the IDs to include` {context}`.